### DISCUSSION TITLE
Registered data with EN through email, why does it not appear in my node OpenSearch Registry?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/registry#265] [Q&A] [answered]
(from a user)

We recently submitted two Voyager Jupiter radio science bundles to EN through Richard Chen. They are searchable via e.g. https://pds.nasa.gov/datasearch/keyword-search/search.jsp?q=voyager+radio+occultation

But I don't see the data in the Registry:

```
% curl -u username 'https://search-rms-prod-hgkpolys7ww6cdoogbx5gsfy6m.us-west-2.es.amazonaws.com/registry/_search?q={_id:"urn:nasa:pds:voyager2_rss_jupiter_raw::1.0"}' | json_pp 

{
   "_shards" : {
      "failed" : 0,
      "skipped" : 0,
      "successful" : 3,
      "total" : 3
   },
   "hits" : {
      "hits" : [],
      "max_score" : null,
      "total" : {
         "relation" : "eq",
         "value" : 0
      }
   },
   "timed_out" : false,
   "took" : 3
}
```

Is it still necessary for us to run harvest and registry-manager on those two bundles?

**Comment by pds-ops** [comment, marked as answer]
Sorry for the confusion here. For the foreseeable future we have 2 “registry systems" running in parallel:

1. **Legacy Registry** – This is the system that drives the [PDS Keyword Search](https://pds.nasa.gov/datasearch/keyword-search/), and other integrated systems. When a node follows the [Data Release Procedure](https://pds.nasa.gov/datastandards/documents/policy/PDS_Release_Announcement_Procedure_v1.0.pdf), and sends an email to [pdsen-dr@jpl.nasa.gov](mailto:pdsen-dr@jpl.nasa.gov) with a release announcement, this is the database that gets loaded with information. This will eventually be deprecated, but will be around for the foreseeable future.
  a. _Node Responsibility_: Follow [Data Release Procedure](https://pds.nasa.gov/datastandards/documents/policy/PDS_Release_Announcement_Procedure_v1.0.pdf)
  b. _EN Responsibility_: Load data

2. **Registry** aka **Next-gen Registry** – This is the system that drives the [PDS Search API](https://pds.nasa.gov/api/search/1/), [pds-deep-registry-archive utility](https://nasa-pds.github.io/deep-archive/usage/index.html#example-3-generating-deep-archives-of-remote-data-in-a-pds-registry), and all future systems.
  a. _Node Responsibility_: Load data using [Harvest and Registry Manager](https://nasa-pds.github.io/registry/user/load1.html)
  b. _EN Responsibility_: Automated utility execution to enhance metadata for provenance, ancestry, data repair, etc. (see [Registry Sweepers](https://github.com/NASA-PDS/registry-sweepers/) for more details)

**Reply by mace-space** [reply]
I cannot send email to pds-dr@jpl.nasa.gov, is this the correct address?

**Reply by jordanpadams** [reply]
@mace-space thanks for catching this. updated the steps to include a link to the data release procedure approved by the MC some time ago, as well as the correct email address

**Reply by mace-space** [reply]
Thanks
